test/data/blue_robotics_t500-command_24V.csv

```diff
@@ -0,0 +1,201 @@
+-10.31,1100

```

Comment:
Where did you get this table from? I am a little out of the loop on this

Reply:
https://drive.google.com/file/d/15D5Aue6qwYzUYzVKHSTL7Bal2g0bCgxY/view?usp=sharing